Allow user reselect photos from gdrive after deleting all rows from table

sonegillis opened the pull request:
When the user select photos from gdrive and later deletes all of them, he still remains on thesame step with the button "Upload to wikimedia commons" and when he clicks on it, an empty filelist is uploaded. Below is an image to show what is explained above
![gcommons-1](https://user-images.githubusercontent.com/22874347/76949308-9fceb380-6908-11ea-94b5-fbbca41a743a.PNG)

The following are steps taken to solve the issue above
- Display to user the view to select photos from google drive when user clears the table with picked files
- Edit the call to gapi auth loader to occur only when the oauthToken is not defined to avoid the auth loader called again

tonythomas01 commented on uploader/wiki_uploader.py:
```diff
@@ -27,13 +27,18 @@ def upload_file(self, file_name, file_stream, description=""):
+            )
+        except mwclient.errors.APIError as e:
+            if e.args[0] == "fileexists-no-change":
+                return False, {}
```
okey, one more thing. If we have an error, lets just return it regardless of the exception type. Right now, the client do not have any idea if the upload failed. A simple side alert on the frontend should be enough I guess with the right message.

sonegillis commented:
Ok, thanks for the review. I'm on it

sonegillis commented:
Hello @tonythomas01, I have done the changes as you requested. I have also added a modal to display the files that weren't uploaded and the reason for the failed upload. For example I added these photos from my google drive as shown in this image
![error-msg-3](https://user-images.githubusercontent.com/22874347/77834013-c5965c80-7141-11ea-8d74-a58c5f5ee4a4.png)
And after clicking the upload button this is the result
![error-message](https://user-images.githubusercontent.com/22874347/77834023-d3e47880-7141-11ea-98b3-9a242391f353.png)
We can see that images that weren't uploaded displayed in the modal with the error message for each message while the successful ones are displayed as normal
